slowquant/qiskit_interface/linear_response/allprojected.py

```diff
@@ -111,21 +111,21 @@ def run(
                     (GI.dagger * GJ).get_folded_operator(*self.orbs)
                 )
                 val -= GG_exp * self.wf.energy_elec
-                val -= self.G_exp[i] * HG_exp[j]
-                val += self.G_exp[i] * self.G_exp[j] * self.wf.energy_elec
+                val -= self._G_exp[i] * self._HG_exp[j]
+                val += self._G_exp[i] * self._G_exp[j] * self.wf.energy_elec
                 self.A[i + idx_shift, j + idx_shift] = self.A[j + idx_shift, i + idx_shift] = val
                 # Make B
-                val = HG_exp[i] * self.G_exp[j]
-                val -= self.G_exp[i] * self.G_exp[j] * self.wf.energy_elec
+                val = self._HG_exp[i] * self._G_exp[j]
+                val -= self._G_exp[i] * self._G_exp[j] * self.wf.energy_elec
                 self.B[i + idx_shift, j + idx_shift] = self.B[j + idx_shift, i + idx_shift] = val
                 # Make Sigma
                 self.Sigma[i + idx_shift, j + idx_shift] = self.Sigma[j + idx_shift, i + idx_shift] = (
-                    GG_exp - (self.G_exp[i] * self.G_exp[j])
+                    GG_exp - (self._G_exp[i] * self._G_exp[j])
                 )
 
     def _get_qbitmap(
         self,
-        cliques: bool = True,
+        cliques: bool = False,
```

Comment:
Yes. But it is not really that relevant. 
I changed it to False because I want to analyse how many CBS strings there actually are for each matrix element in LR. Just having the clique head does not give this number correctly. In the end, if we look at a whole matrix element and if this is fully in CBS or not, it does not matter. And (in the end)² this analysis is kind of outdated with the new variance analysis